entsoe-util/src/test/java/com/powsybl/entsoe/util/BoundaryPointXlsParserTest.java

```diff
@@ -63,4 +70,21 @@ public void test() throws IOException {
         assertEquals(Country.FR, point.getBorderFrom());
         assertEquals(Country.BE, point.getBorderTo());
     }
+
+    @Test
+    public void testMissingBoundaryPointFile() {
+        BoundaryPointXlsParser parser = new BoundaryPointXlsParser();
+
+        PlatformConfig mockedPc = Mockito.mock(PlatformConfig.class);
+        Mockito.when(mockedPc.getConfigDir()).thenReturn(Optional.empty());
+
+        try (MockedStatic<PlatformConfig> staticMockedPc = Mockito.mockStatic(PlatformConfig.class)) {
```

Comment:
Why do you need this static mock ? There is already a way to mock platform config for test.
Is unit tests are still thread safe using static mocks?

Reply:
It was only for coverage reason. The `Mockito.mockStatic` is only for current thread:

> Creates a thread-local mock controller for all static methods of the given class or interface.


I'll remove it in this test, as the corresponding coverage is not worth it. It was maybe more interesting in `EmptyPlatformConfigTest`, but the test is a bit twisted after all: it is mocking `ServiceProvider::load` and `ServiceProvider::iterator` to test `PlatformConfig::defaultConfig`.